xpano/gui/panels/about.cc

```diff
@@ -14,6 +14,12 @@ namespace xpano::gui {
 
 namespace {
 
+template <typename TType>
+bool IsReady(const std::future<TType>& future) {
```

Comment:
Please move this function to `xpano/utils/future.h` to namespace xpano::utils::future.

Call it from both here and pano_gui.cc

Reply:
I couldn't find the file `xpano/utils/future.h`. Do I have to create the file myself?